central/notifier/service/service_impl_test.go

```diff
@@ -18,6 +19,10 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+func TestAuthz(t *testing.T) {
+	testutils.AssertAuthzWorks(t, &serviceImpl{})
+}
+
```

Comment:
Bit surprised by this change, not sure I'm missing context?

Reply:
Eh damn /s/notifier/notifications.